tests/boot/boot_to_desktop.pm

```diff
@@ -24,7 +24,9 @@ sub run {
         wait_serial("Welcome to SUSE Linux", $timeout) || die "System did not boot in $timeout seconds.";
     }
     else {
-        $self->wait_boot(bootloader_time => $timeout);
+        # live may take ages to boot
+        my $ready_time = get_var('LIVETEST') ? 700 : undef;
```

Comment:
@DrMullings with setting to undef we are falling back to the default ready_time which is set within wait_boot. With this we can preserve the behaviour that the previous L27 had with not specifying the ready_time